src/api/app/views/webui/project/keys_and_certificates.html.haml

```diff
@@ -0,0 +1,56 @@
+- @pagetitle = "Key and certificate for #{@project}"
+
+.card.mb-3
+  = render partial: 'tabs', locals: { project: @project }
```

Comment:
You render tabs but have no entry in it. Is this intentional?

Reply:
Yes, I want to signal that we're still in the "same" place than before, but we have no entry in the tab.

Reply:
Then this should make `Overview` active right?

Reply:
Right

Reply:
Done